docs/recipes/pug.md

```diff
@@ -34,27 +34,47 @@ $ npm install --save-dev gulp-pug
 Add this task to your `gulpfile.js`, it will compile `.pug` files to `.html` files in `.tmp`:
 
 ```js
-gulp.task('views', () => {
-  return gulp.src('app/*.pug')
+function pug() {
+  return src('app/*.pug')
     .pipe($.plumber())
     .pipe($.pug({pretty: true}))
-    .pipe(gulp.dest('.tmp'))
-    .pipe(reload({stream: true}));
-});
+    .pipe(dest('.tmp'))
+    .pipe(server.reload({stream: true}));
+}
 ```
 
 We are passing `pretty: true` as an option to get a nice HTML output, otherwise Pug would output the HTML on a single line, which would break our comment blocks for wiredep and useref.
```

Comment:
We're no longer using wiredep:

```suggestion
We are passing `pretty: true` as an option to get a nice HTML output, otherwise Pug would output the HTML on a single line, which would break our comment blocks for useref.
```